executor-service/src/main/java/executor/service/service/step/UnsupportedStepExecution.java

```diff
@@ -0,0 +1,23 @@
+package executor.service.service.step;
+
+import executor.service.exception.StepExecutionException;
+import executor.service.model.Step;
+import org.openqa.selenium.WebDriver;
+
+import static executor.service.service.executor.Action.UNSUPPORTED_ACTION;
+
+public class UnsupportedStepExecution
```

Comment:
Do we really need a separate StepExecution class for unsupported actions, whose method just throws an exception?

Reply:
Good point, keep it short and simple